packages/calcite-components/conventions/README.md

```diff
@@ -142,6 +142,10 @@ There are a few ways to add event listeners within our components:
 
 Private/internal properties should be annotated accordingly to avoid exposing them in the doc and/or API. You can do this by using the `@private`/`@internal` [JSDoc](https://jsdoc.app/) tags.
 
+### Property values
+
+Property values should be meaningful. Avoid using values such as "default" for default values. Instead, use a more descriptive value. 
```

Comment:
Great addition! 📝 

nitpick:
Is it possible to provide an example of what the default could look like vs. what could be implemented instead?
> For instance: `highlightPlacement: "start"` over `highlightMode: "default"`

Reply:
Updated!